PlanGrid.Api/AnnotationVisibility.cs

```diff
@@ -0,0 +1,19 @@
+﻿// <copyright file="AnnotationVisibility.cs" company="PlanGrid, Inc.">
```

Comment:
Might be worth adding a post-build time check for this (you can copy what we have in PG.exe: `.\Build\CopyrightCheck.ps1`)
When someone else works on this and forgets to/doesn't have ReSharper, etc.